README.md

```diff
@@ -1 +1,77 @@
-# hello-microverse
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# hello-microverse
+
+> This is a site showing hello microverse text as a header.
+
+![screenshot](./images/pic.png)
+
+It has a white background with the header in red color.
+
+## Built With
+
+- html,css
+- Linters,Mark down
+
+
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+1)Enter `git clone git@github.com:basnetlaxmi/hello-microverse.git` into terminal
+
+2)Open in browser to view site
+
+
+### Prerequisites
+-code editor
+-git and github
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```

Comment:
You should leave only one author in this solo project, and please add the links for your own social media.

- [ ] Remove extra author information.
- [ ] Add your own social media links in this section of your README file.